components/LockScreen.tsx

```diff
@@ -54,13 +120,30 @@ export function LockScreen() {
     >
       <View style={styles.content}>
         {lockMode === 'pin' && (
-          <PinInput
-            title={t('lockScreen.enterPin')}
-            subtitle={t('lockScreen.enterPinSubtitle')}
-            onPinComplete={handlePinComplete}
-            errorMessage={errorMessage}
-            onStartTyping={handleStartTyping}
-          />
+          <>
+            {isLockedOut ? (
+              <View style={styles.lockoutContainer}>
+                <Text style={[styles.lockoutTitle, { color: colors.error }]}>
+                  {t('lockScreen.lockedOut')}
+                </Text>
+                <Text style={[styles.lockoutMessage, { color: colors.textSecondary }]}>
+                  {t('lockScreen.lockedOutMessage')}
+                </Text>
+                <Text style={[styles.lockoutTimer, { color: colors.textPrimary }]}>
+                  {remainingTime}
+                </Text>
+              </View>
+            ) : (
+              <PinInput
+                title={t('lockScreen.enterPin')}
+                subtitle={t('lockScreen.enterPinSubtitle')}
+                onPinComplete={handlePinComplete}
+                errorMessage={errorMessage}
+                onStartTyping={handleStartTyping}
+                disabled={isPinDisabled}
+              />
+            )}
+          </>
         )}
         {lockMode === 'biometric' && (
```

Comment:
So you're only protecting PIN locks but not biometrics? I don't see why the specific method (PIN vs biometrics) matters -- if the app should be locked, it should be locked regardless.